fedmsg_meta_fedora_infrastructure/tests/buildsys.py

```diff
@@ -310,7 +348,7 @@ class TestKojiBuildStateChangeStartNoOwner(Base):
     }
 
 
-class TestKojiBuildStateChangeFail(Base):
```

Comment:
Would it make sense to include 'cancel' or 'fail' as they are likely going to have different info compared to 'complete' (I'm thinking especially about the long_form).

Note: I won't block the PR on this, it can always be added later if we wish to :)